apps/export/entries/report_exporter.py

```diff
@@ -459,29 +453,24 @@ def _generate_for_entry_widget_data(self, entry, para):
         if entry.id not in self.entry_widget_data_cache:
             raw_export_data = []
             for each in entry.exportdata_set.all():
-                if 'other' in each.data.get('report', {}):
-                    for rep in each.data['report'].get('other', []):
-                        if rep.get('widget_key') and rep['widget_key'] in self.exporting_widgets_keys:
-                            raw_export_data.append(rep)
-                else:
-                    export_datum = {
-                        **each.data.get('common', {}),
-                        **each.data.get('report', {}),
-                    }
-                    if export_datum.get('widget_key') and export_datum['widget_key'] in self.exporting_widgets_keys:
-                        raw_export_data.append(export_datum)
+                export_datum = {
+                    **each.data.get('common', {}),
```

Comment:
I think `data.get('common') or {}` will be safer here, just in case that the value is deliberately set to `None`. If it is guaranteed that it cannot happen, this should be just fine.

Reply:
Sure. Done. There are lots of these though. Maybe we should just use [pydash](https://pydash.readthedocs.io/en/latest/) `pydash.get(`.